website/docs/reference/dbt-commands.md

```diff
@@ -99,3 +124,50 @@ Use the following dbt commands in [dbt Core](/docs/core/installation-overview) a
 
 </Tabs>
 </VersionBlock>
+
+### Execute commands
+
+When you're managing your dbt project, it's important to understand that dbt commands are categorized into the following types:
+
+- **Write commands** &mdash; These commands perform actions that modify data or metadata in your data platform.
+- **Read commands** &mdash; These commands involve operations that fetch or read data without making any modifications to your data platform.
+
+You can run multiple dbt commands at the same time, but it's important to understand which commands can be run in parallel and which can't. We call this 'parallel execution' and it's important when distinguishing between write and read commands:
+
+- Write commands &mdash; Commands such as `dbt build` and `dbt run` are limited to one invocation at any given time (1 parallelism). This is to prevent any potential conflicts, such as overwriting the same table in your data platform, at the same time.
+- Read commands &mdash; Commands such as `dbt parse` and `dbt source snapshot-freshness` can have multiple invocations in parallel and aren't limited to one invocation at any given time. This means read commands can run in parallel with both other read commands and write commands.
+
+To ensure your dbt workflows are both efficient and safe, you can run different types of dbt commands at the same time (in parallel). For example, `dbt build` (write operation) can safely run alongside `dbt parse` (read operation) at the same time. However, you can't run `dbt build` and `dbt run` (both write operations) at the same time.
+
+For a detailed view of which commands can execute in parallel, expand and check out the following table:
+<expandable alt_header="Detailed table of parallel execution commands.">
+
+The following table highlights write or read commands and whether you can execute them in parallel with each other. 
+
+Note that some dbt commands aren't listed in the following table since they're not relevant to the parallelization of dbt commands.
+
+| Command          | Type       | Parallel execution <br /> with other commands |
+| :-----------------: | :----------:|:------------------:|
+| `build`          | Write  |     ❌             |
+| `clone`          | Write  |     ❌              |
+| `retry`          | Write  |     ❌             |
+| `run`            | Write  |     ❌             |
+| `run-operation`  | Write  |     ❌             |
+| `seed`           | Write  |     ❌             |
+| `snapshot`       | Write  |     ❌             |
+| `clean`          | Read |      ✅           |
+| `compile`        | Read |      ✅           |
```

Comment:
but i tested compile multiple times and found it also gives adapter error.

Reply:
Hi @mirnawong1  i mentioned the issue here: 
-> Issue i have metioned different commands 
-> But also I tried multiple compile comands which give the same KeyError

Reply:
@hash-data @mirnawong1 This is why it's important we make a distinction between what's supported with:
- dbt-core (CLI or programmatic invocation), which does not support concurrency within the same process (but it's possible to achieve this with additional work)
- Cloud CLI, which does that additional work to manage concurrency (multiple processes) on the user's behalf